js/poloniex.js

```diff
@@ -549,8 +549,8 @@ module.exports = class poloniex extends Exchange {
         return this.filterBySinceLimit (result, since, limit);
     }
 
-    async fetchOrder (id, symbol = undefined, params = {}) {
-        let orders = await this.fetchOrders (symbol, params);
+    async fetchOrder (id, symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        let orders = await this.fetchOrders (symbol, since, limit, params);
```

Comment:
Hi! Unfortunately, we can't do that, it will break fetchOrder compatibility with other exchanges.

Reply:
Hello,
Just calling fetchOrders(symbol, undefined, undefined, params)  would be acceptable ?

Reply:
Yep!

Reply:
OK gonna push a new commit using params.limit and params.since